geode-core/src/main/java/org/apache/geode/pdx/internal/PeerTypeRegistration.java

```diff
@@ -740,6 +736,7 @@ public EnumInfo getEnumById(int id) {
   private void updateClassToTypeMap(PdxType type) {
```

Comment:
this method is called on afterCreate on a CacheListener. I think this listener should also update the enumToId map but it currently does not.

Reply:
enum will be fixed in another ticket.

Reply:
What is the ticket number?

Reply:
GEODE-7035